### ISSUE TITLE
How to do an exact phrase search?

### ISSUE THREAD

**Issue by bsabalaskey** [10up/ElasticPress#3232]
### Describe your question

I am trying to do an exact phrase search, that is with multiple words with obvious spaces between them. In my theme, I detect if the search has double quotes, like "agenda 2030". Then I hook into the ep_post_formatted_args_query filter and create a query with the keyword analyzer and the of phrase_prefix, as show below from my data dump. I also removed the queries for cross_fields and the keyword multimatch with the 'and' operator.

```
Array
(
    [bool] => Array
        (
            [should] => Array
                (
                    [0] => Array
                        (
                            [multi_match] => Array
                                (
                                    [query] => agenda 2030
                                    [type] => phrase_prefix
                                    [fields] => Array
                                        (
                                            [0] => post_title
                                            [1] => post_excerpt
                                            [2] => post_content
                                            [3] => terms.post_tag.name^3
                                            [4] => terms.category.name^3
                                        )
                                    [boost] => 3
                                    [analyzer] => keyword
                                    [fuzziness] => 0
                                )
                        )
                )
        )
)
```

However, matches are returned which do not have the phrase but just both keywords 'agenda' and '2030'.

How can I get an exact phrase search to match?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felipeelia** [comment]
Hi @bsabalaskey, I'm not exactly sure why that is not working but in the meantime, did you already try using [ElasticPress Labs](https://github.com/10up/ElasticPressLabs)? The "Boolean Search Operators" feature adds support to double-quoted terms and you wouldn't have to make any changes in your code.
Can you give that a try and let us know if it works for you? Thanks!

**Comment by bsabalaskey** [comment]
I found the primary problem - I mistakenly used the "fuzziness" parameter, which caused my ES server to return a 400 error, which then caused the default WP search to be used. So I removed the fuzziness option and progress was made. However the analyzer of "keyword" returned nothing at all. I then tried using the analyzer "standard" or "whitespace" (and lower casing the search terms) which worked. This can be closed.
